index.js

```diff
@@ -1,12 +1,13 @@
 'use strict';
 
-const broccoliFeatures = Object.freeze({
+const PATHS = new WeakMap();
+
+const BROCCOLI_FEATURES = Object.freeze({
   persistentOutputFlag: true,
   sourceDirectories: true,
   needsCacheFlag: true,
   volatileFlag: true,
 });
-const PATHS = new WeakMap();
```

Comment:
Removing this seems like a bad rebase? I don't see the usage of `PATHS` being removed below

Reply:
Ya... https://github.com/broccolijs/broccoli-plugin/pull/37#discussion_r302810494